packages/teams-js/src/artifactsForCDN/validDomains.json

```diff
@@ -5,20 +5,20 @@
     "gov.teams.microsoft.us",
     "dod.teams.microsoft.us",
     "int.teams.microsoft.com",
-    "teams.live.com",
-    "devspaces.skype.com",
-    "ssauth.skype.com",
-    "local.teams.live.com",
-    "local.teams.live.com:8080",
-    "local.teams.office.com",
-    "local.teams.office.com:8080",
+    "*.teams.microsoft.com",
     "outlook.office.com",
     "outlook-sdf.office.com",
     "outlook.office365.com",
     "outlook-sdf.office365.com",
     "outlook.live.com",
     "outlook-sdf.live.com",
-    "*.teams.microsoft.com",
+    "teams.live.com",
```

Comment:
>"teams.live.com", [](http://example.com/codeflow?start=4&length=17)

Since you are grouping them, can you put grouping comments into here so others know how they are grouped? Otherwise it may just look unorganized and people will throw stuff at the bottom. E.g.,

```
{
  "validOrigins": [
    // Teams
    "teams.microsoft.com",
    "teams.microsoft.us",
    "gov.teams.microsoft.us",
    "dod.teams.microsoft.us",
    "int.teams.microsoft.com",
    "*.teams.microsoft.com",
    // Outlook
    "outlook.office.com",
    "outlook-sdf.office.com",
    "outlook.office365.com",
    "outlook-sdf.office365.com",
    "outlook.live.com",
    "outlook-sdf.live.com",
    // Office / Microsoft 365
```
?

Reply:
I tried to do that, but VS code complained about comments not being allowed in JSON

Reply:
Bummer